bin/desi_archive_tilenight

```diff
@@ -117,33 +122,58 @@ else:
     reduxdir = args.prod
 
 if args.specstatus is None:
-    args.specstatus = os.path.expandvars('$DESI_ROOT/survey/ops/surveyops/trunk/ops/tiles-specstatus.ecsv')
+    if os.path.exists('./tiles-specstatus.ecsv'):
```

Comment:
this is logged a few lines later just before reading the file:
```
log.info(f'Reading tiles from {args.specstatus}')
```

Reply:
Thanks, I had overlooked that. My thought was that it may be useful to tell the user that you found a copy in the current dir and were therefore using it, but they would likely piece it together given the path of the file used.